### ISSUE TITLE
Rendered Columns do not work with Serverside Tables

### ISSUE THREAD

**Issue by JustinGrote** [ironmansoftware/powershell-universal-legacy#76]
#### Description

When creating a serverside table, if I have an extra custom column and attempt to render it with `New-UDTableColumn -Render`, it fails with 'undefined'

![image](https://user-images.githubusercontent.com/15258962/97644880-b1347e00-1a08-11eb-93c0-d6c8bf14b29c.png)

#### Additional Information
TruncatedExample
```powershell
            $UDTableParams = @{
                #Data = Get-PasswordLastSetUsers
                LoadData = {
                    #BUG: Remove when https://github.com/ironmansoftware/powershell-universal/issues/75 is resolved
                    $EventData.properties = $null
                    $EventData.properties = ($Body | ConvertFrom-Json).properties
                    Format-SDMUserTable (Get-PasswordLastSetUsers) -TableData $EventData
                }
                Columns = @(
                    New-UDTableColumn -Property SamAccountName -Title "User"
                    New-UDTableColumn -Property Name -Title "Display Name"
                    New-UDTableColumn -Property PasswordLastSet -Title "Password Last Set"
                    New-UDTableColumn -Property Unlock -Title "Action" -Render {
                        $buttonId = "PWDResetBtn-" + $eventData.SamAccountName
                        $buttonAction = {
                            Reset-SDMUserPassword -SamAccountName $args[0].SamAccountName -ErrorAction Stop
                        }
                        New-UDButton -Id $buttonId -Icon (New-UDIcon -Icon key) -Text 'Reset' -OnClick (
                            New-UDEndpoint -Endpoint $DoButtonAction -ArgumentList @(
                                $buttonId,
                                "Resetting Password for $($eventData.SamAccountName):",
                                $buttonAction,
                                $eventData
                            )
                        )
                    }
                )
            }
```

#### Version Information

- Universal: 1.4.6
- Universal Hosting Method (IIS, Executable, docker etc): Any
- PowerShell: Any
- Operating System: Any

#### Internal Issue

We'll fill this out when we create an internal issue in our repository.

**Comment by adamdriscoll** [comment]
I couldn't reproduce this one but maybe it had something to do with the other things I fixed? One thing I noticed in your example is you are defining `$buttonAction` but then using a variable called `$DoButtonAction` in `New-UDEndpoint`. 

I'm going to close this since it appears to be resolved but if you still encounter this in 1.4.7, please feel free to reopen.

**Comment by JustinGrote** [comment]
@adamdriscoll its a snippet out of a larger codebase but I'll try again on 1.4.7 and report more detail if still an issue. Thanks!
